pkg/systemd/logs.scss

```diff
@@ -59,15 +59,67 @@
     margin-top: 4px;
 }
 
-#journal-current-day-menu dropdown-toggle {
-    padding-left: 10px;
+#journal .filter select,
+#journal .filter input {
+    margin-right: 20px;
+    margin-top: 2px;
+    margin-bottom: 2px;
+}
+
+#journal-box .panel-heading {
+    position: sticky;
+    top: 57px;
+}
+
+.filter {
+    display: inline-block;
 }
 
 .content-header-extra + #journal-box {
     margin: 0;
     padding: 1.5rem;
 }
 
+.content-header-extra {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+}
+
+.content-header-extra.toggle-filters-closed .filter:not(:first-child) {
+    display: none;
+}
+
+.filters-toggle {
+    flex-grow: 1;
+}
+
+#journal-service-menu {
+    width: 10em; // Static width as it likes to resize a lot while loading
+}
+
+.filters-toggle button {
+    float: right;
+}
+
+.full-row {
+    flex-basis: 100%;
+    height: 0;
+}
+
+.filter.text-search {
+    min-width: 150px;
+    max-width: 700px;
```

Comment:
What is the reason for this line?
I think it's OK that the filter takes as much space as it wants.

Also makes the pause button not line up with the list below in some situations.
![Screenshot from 2020-04-23 14-29-32](https://user-images.githubusercontent.com/209895/80099743-409b3900-856f-11ea-8507-c36c565014ab.png)

Reply:
> What is the reason for this line?

Good point. Not much any more. I had it there when the screen was not wide enough to have things on one line, but wide enough that the search was on separate line it would span too much. But now it does nothing really. I'll remove it.

Reply:
Done